test/src/main/java/io/strimzi/test/k8s/KubeClient.java

```diff
@@ -232,6 +232,14 @@ public List<String> listPodNamesInSpecificNamespace(String namespaceName, String
             .collect(Collectors.toList());
     }
 
+    public PersistentVolumeClaim getPVC(String namespaceName, String pvcName) {
```

Comment:
Camel case and same for delete
```suggestion
    public PersistentVolumeClaim getPvc(String namespaceName, String pvcName) {
```

Reply:
Why camel case here? It's an abbreviation for `PersistentVolumeClaim`. For `StrimziPodSet` it's also SPS.